packages/dashboard/src/utils/useStoryView.js

```diff
@@ -116,16 +120,45 @@ export default function useStoryView({
     setPageClamped(page + 1);
   }, [page, setPageClamped]);
 
+  const toggleAuthorFilterId = useCallback(({ id }) => {
+    _setAuthorFilterId((v) => (v === id ? null : id));
```

Comment:
💭 maybe we could name this something more descriptive
```suggestion
    _setAuthorFilterId((prevFilterId) => (prevFilterId === id ? null : id));
```